conda-envs/mlos-3.10.yml

```diff
@@ -10,8 +10,7 @@ dependencies:
   - pycodestyle
   - pydocstyle
   - flake8
-  - setuptools
-  - setuptools-scm>=8.1.0
+  - build
```

Comment:
`build` can pull `setuptools-scm` and `setuptools` for us now via `pyproject.toml`